core/src/main/java/tachyon/web/UiFileInfo.java

```diff
@@ -119,7 +127,11 @@ public String getCheckpointPath() {
   }
 
   public String getCreationTime() {
-    return CommonUtils.convertMsToDate(mCreationTimeMs);
+    if (mCreationTimeMs == LocalFileInfo.EMPTY_CREATION_TIME) {
+      return " ";
```

Comment:
Probably not, but I was just staying consistent with getBlockSizeBytes() and getSize(). Should I just change them all?